src/components/login/container/LoginDetails.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../styles/index.css';
+import newVideo from '../styles/video/newInstructions.mp4';
+
+const LoginDetails = () => (
+  <div className="login-details">
+    <div className="card">
+      <div className="card-header">
+        <strong>What to ensure</strong>
+      </div>
+      <div className="card-body">
+        Ensure you have an account from the traditional OCL. After that,{' '}
+        <a href="https://qa.openconceptlab.org/accounts/login/">
+          Login into the traditional OCL
```

Comment:
I do not think using the _a_ tag is ideal as it would cause a page to reload which goes against the essence of React, I'd advise you use the Link component provided by react-router-dom.

Reply:
ok thanks, will do that

Reply:
Hey @emasys also commented on this and actually recommend to change it back to `a` so that's why its back

Reply:
This is an external link, you might want to make it open in a new tab

Reply:
After thorough research and discussions, we decided to leave the ```a``` for external links, therefore this comment was resolved.

Reply:
Does this mean we shall again change from qa.openconceptlab.org back to https://openconceptlab.org?

Reply:
What do others think?

Reply:
I also agree with @emasys because if we are to change the ```qa``` even the API endpoint needs to change, It looks like a ticket on its own

Reply:
Can u create the ticket?